tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -354,18 +350,6 @@ def copy_vocab_tables(self, vocabulary_id):
                                                      exists_ok=True)
                 self.client.copy_table(src_table, dst_table)
 
-                self.vocabulary_tables.append(destination)
-
-            self.vocabulary_loaded = True
-
-        def tearDown(self):
-            super().tearDown()
-
-            # Remove all vocabulary tables if created
-            if self.vocabulary_loaded:
-                for table in self.vocabulary_tables:
-                    self.client.delete_table(table, not_found_ok=True)
-
```

Comment:
@jp3477 It's good to revert this commit, but I like your thought process.  If we are going to add the vocabulary tables with the base class, we can add them to the "automatic cleanup" at this time as well.  But I think instead of all of this, you could say:
```python
self.client.copy_table(src_table, dst_table)    # this currently exists in copy_vocab_tables
self.fq_table_names.append(destination)     # this could be added
```
This should add them to the list of names that will be deleted by either the class or test `tearDown` functions.  Both say `not_found_ok=True` so if something is added twice, it isn't a failure.

Reply:
@jp3477 Are you interested in adding this line to the `copy_vocab_tables` method?

Reply:
@lrwb-aou Sure, I'll add this in. Think it'll be useful